src/OSPSuite.Core/Domain/ModelCoreSimulationExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace OSPSuite.Core.Domain
+{
+   public static class ModelCoreSimulationExtensions
```

Comment:
I decided to create extension methods to ensure that more complex code won;'t be duplicated between pksim and mobi (a PKSim simulation inherits from PKSimBuildingBlock and as such won't be able to inherit ModelCoreSimulation)